packages/beacon-node/src/chain/lightClient/index.ts

```diff
@@ -291,6 +293,31 @@ export class LightClientServer {
     return update;
   }
 
+  /**
+   * API ROUTE to get the current sync committee from the best available update for `period`.
```

Comment:
minor: the comment here is semantically different from the corresponding comment on api route